loans/src/main/java/com/microsevicesbank/loan/controller/LoanController.java

```diff
@@ -26,6 +32,30 @@ public class LoanController {
 
     private ILoanService loanService;
 
+    @Operation(
+            summary = "Create new loan REST API",
+            description = "Creates a new loan based on provided 10-digit mobile number"
+    )
+    @ApiResponses({
+            @ApiResponse(
+                    responseCode = "201",
+                    description = "HTTP Status 201 CREATED"
+            ),
+            @ApiResponse(
+                    responseCode = "400",
+                    description = "HTTP Status 400 BAD REQUEST",
+                    content = @Content(
+                            schema = @Schema(implementation = ErrorResponseDTO.class)
```

Comment:
In future it will be needed to create **more error responses** to describe an exception more precisely